### ISSUE TITLE
Interface with pyrokinetics

### ISSUE THREAD

**Issue by bpatel2107** [stellaGK/stella#113]
Pyrokinetics has settled down in terms of structure so now would be a good time to interface Stella with it. We are writing several publications based on pyro so it would be nice to get stella in before they are submitted.

**Comment by mrhardman** [comment]
Has any work been attempted on this issue since Aug 2023?

**Comment by bpatel2107** [comment]
I know some work was done in a branch on pyro here 
https://github.com/pyro-kinetics/pyrokinetics/tree/enhancements/gk_code_stella

But I think it's quite out of date. Don't think it would be a huge amount of work to get it up and running. I'm away until March but if someone wanted to get this up and running I can support from then on and the other developers can help too.

**Comment by bpatel2107** [comment]
@mrhardman Happy to get back into this if you are keen

**Comment by mrhardman** [comment]
@bpatel2107 Yes it would be good to book some time in to look at this soon. We are looking to benchmark stella against other reliable codes for EM simulations, so now could be an ideal time. I'll send an email directly.
